x-pack/solutions/security/plugins/security_solution/public/plugin.tsx

```diff
@@ -191,8 +217,7 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
       hideFromSidebar: true,
       order: 1,
       mount: async (params) => {
-        const { renderApp, services, store } = await mountDependencies();
-        const { ManagementSettings } = await this.lazyAssistantSettingsManagement();
```

Comment:
Not part of fix but I saw this while in the file. This creates a waterfall of chunk loading. I moved `ManagementSettings` chunk loading into `mountDependencies` to avoid this waterfall.